draft-ietf-quic-http.md

```diff
@@ -261,16 +261,23 @@ the stream management.
 All client-initiated bidirectional streams are used for HTTP requests and
 responses.  A bidirectional stream ensures that the response can be readily
 correlated with the request. This means that the client's first request occurs
-on QUIC stream 0, with subsequent requests on stream 4, 8, and so on. HTTP/QUIC
-does not use server-initiated bidirectional streams. The use of unidirectional
-streams is discussed in {{unidirectional-streams}}.
+on QUIC stream 0, with subsequent requests on stream 4, 8, and so on. In order
+to permit these streams to open, an HTTP/QUIC client SHOULD send non-zero values
+for the QUIC transport parameters `initial_max_stream_data_bidi_local`. An
+HTTP/QUIC server SHOULD send non-zero values for the QUIC transport parameters
+`initial_max_stream_data_bidi_remote` and `initial_max_bidi_streams`.
 
 These streams carry frames related to the request/response (see {{frames}}).
 When a stream terminates cleanly, if the last frame on the stream was truncated,
 this MUST be treated as a connection error (see HTTP_MALFORMED_FRAME in
 {{http-error-codes}}).  Streams which terminate abruptly may be reset at any
 point in the frame.
 
+HTTP/QUIC does not use server-initiated bidirectional streams. The use of
+unidirectional streams is discussed in {{unidirectional-streams}}.  Both clients
+and servers SHOULD send a non-zero value for the QUIC transport parameter
+`initial_max_uni_streams`.
```

Comment:
Would it make sense to state recommend "at least three", considering the fact that there would be the controller stream and the QPACK encoder / decoder streams?